### PR TITLE
added price filter for /locations

### PR DESCRIPTION
**Pull request by kevo1ution**

### CODE REVIEW

File: src/api/routes/location.js [hunk 1]
```diff
@@ -18,6 +18,15 @@ router.get('/',
         radius: Joi.number().max(MAX_DISTANCE_METERS).min(0).required(),
         veganRating: Joi.number().valid(3.5, 4.0, 4.5),
         open: Joi.number().integer().max(7 * 24 * 60).min(0),
+        price: Joi.string().max('$,$$,$$$,$$$$'.length).custom((value, helpers) => {
+          for (let i = 0; i < value.length; i += 1) {
```
Comment (sarahannali) [inline src/api/routes/location.js:22]: why not i++

File: src/api/routes/location.js [hunk 2]
```diff
@@ -18,6 +18,15 @@ router.get('/',
...
+            if (!(value[i] === '$' || value[i] === ',')) {
+              return helpers.error('any.invalid');
+            }
+          }
+
+          return value.split(',');
```
Comment (sarahannali) [inline src/api/routes/location.js:28]: why not just split as you do the error checking

File: src/api/routes/location.js (hunk 1 above)
Comment (kevo1ution) [reply src/api/routes/location.js:22]: eslint won't let me

File: src/api/routes/location.js (hunk 2 above)
Comment (kevo1ution) [reply src/api/routes/location.js:28]: using built-in split is less error-prone, and not worried about performance here

**Review by sarahannali — APPROVED** [review]

**Pull request merged**